tools/package.json

```diff
@@ -5,14 +5,16 @@
 	"scripts": {
 		"check:type": "tsc",
 		"test:ci": "vitest run",
-		"check:lint": "eslint ."
+		"check:lint": "eslint .",
+		"test:file": "node -r esbuild-register test/run-test-file.ts"
 	},
 	"devDependencies": {
 		"@cloudflare/eslint-config-worker": "workspace:*",
 		"@cloudflare/workers-tsconfig": "workspace:*",
 		"@typescript-eslint/eslint-plugin": "^5.55.0",
 		"@typescript-eslint/parser": "^5.55.0",
 		"ts-dedent": "^2.2.0",
-		"undici": "5.28.4"
+		"undici": "5.28.4",
+		"find-up": "^6.3.0"
```

Comment:
@petebacondarwin I put the dependency in this package because that's what we seem to do elsewhere (install dependencies in the package where they are consumed). Didn't know if this sort of thing we would hoist to the root. I used the same version as we use with `wrangler` so that they would be easier to upgrade together if necessary.

Reply:
That's the right thing to do. The TS file will be executed from inside this package.